.github/workflows/ci.yaml

```diff
@@ -117,6 +167,39 @@ jobs:
       - name: Format
         run: mix format --check-formatted
 
+  Seed:
```

Comment:
Hmm, we should probably not include this for libraries. Will work on updating common-config-elixir for this at some point.